src/api/roles/client/client.py

```
@@ -1198,3 +1198,8 @@ def pay_invoice(invoice_id: int, payload: PayInvoiceInput, db = Depends(get_sess
         amount_paid=payload.amount,
         remaining_balance=invoice.outstanding_balance,
     )
+
+
+@router.get("/dummy", response_model = DunderResponse)
+def get_dummy(db = Depends(get_session), acc: Account = Depends(get_client_account)):
```

Ruff (0.15.12)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)